src/test/resources/Features/OCD-2797-surveillanceUpload.feature

```diff
@@ -0,0 +1,13 @@
+@Regression
+Feature: OCD-2797 - Verify surveillance upload/confirm workflow is functional
+
+  Scenario: Upload and confirm surveillance activity
+    Given I'm logged in as "ROLE_ACB"
+    And I am on Upload Products & Surveillance page
+    When I upload a surveillance activity for listing with CHPL ID 15.05.05.2760.ISCD.01.00.1.181101
```

Comment:
What if this was
```suggestion
    When I upload the "SurveillanceUpload_SLI.csv" surveillance activity
```
and you use the variable to choose between the surveillances that are available? Then, when/if we need a different surveillance for some reason you can re-use the step and add a new csv to the project?